src/routes/saved_objects/saved_object_types/dashboard.ts

```diff
@@ -0,0 +1,36 @@
+import { Range } from "../types";
+
+interface RefreshInterval {
+  display: string;
+  pause: boolean;
+  value: number
+}
+
+type DashboardType = 'dashboard'
+
+export type Dashboard_13_2_0 = {
+  attributes: {
```

Comment:
Please run `GET .siren/_mappings` on dev tools and assign any `Keyword` or `Double` types so we can use them for creating ES mappings

Then add comments to the attributes that are either stringified arrays or stringified JSONs, I think that'll help speed up the ticket where we store these as parsed values